pkg/metric/grafana.go

```diff
@@ -0,0 +1,58 @@
+package metric
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/grafana-tools/sdk"
+)
+
+func CreateMetricsDashboard(apiURL, dashboardName string, metrics map[string]string) error {
+	client := sdk.NewClient(apiURL, "admin:admin", http.DefaultClient)
+	ctx := context.Background()
+	boardFile, err := ioutil.ReadFile("templates/dashboard.json")
```

Comment:
The `metrics-check` is supposed to use everywhere. We'd better put the json content in a go file.

Reply:
Actually, the grafana sdk provides data structure represent dashboards:
https://github.com/grafana-tools/sdk/blob/c831d821bb8a4f5fd39703ffd4ac87ef22de7b90/board.go#L43

This json file is a temporary solution. I don't know how to initialize the `Dashboard` from grafana sdk properly yet.

Reply:
The json may be more simple to maintain, and I think the simple way is to have a string
```golang
dashboardJsonString = `
blabla
`
```

Reply:
Good